chartmogul.go

```diff
@@ -124,7 +124,7 @@ type IApi interface {
 	// Metrics - Subscriptions & Activities
 	MetricsListCustomerSubscriptions(cursor *Cursor, customerUUID string) (*MetricsCustomerSubscriptions, error)
 	MetricsListCustomerActivities(cursor *Cursor, customerUUID string) (*MetricsCustomerActivities, error)
-	MetricsListActivities(cursor *Cursor) (*MetricsActivities, error)
+    MetricsListActivities(MetricsListActivitiesParams *MetricsListActivitiesParams) (*MetricsActivities, error)
```

Comment:
It seems indentation is wrong, perhaps text editor settings?

Reply:
Go fmt did this. I will check again.